activejob/test/cases/translation_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'helper'
+require 'jobs/translated_hello_job'
+
+class TranslationTest < ActiveSupport::TestCase
+  setup do
+    JobBuffer.clear
+    I18n.available_locales = [:en, :de]
+    @job = TranslatedHelloJob.new('Johannes')
+  end
+
+  teardown do
+    I18n.available_locales = [:en]
+  end
+
+  test 'it performs the job in the given locale' do
+    @job.locale = :de
+    @job.perform_now
```

Comment:
@robin850 Thanks for having a look at it. From my point of view the `test 'it performs the job in the given locale' do` does a good job to test this behavior: it serializes, deserializes and executes the job, expecting that the outcome is localized.

Could you outline which test you'd like to see? I'm happy to add it then! :)